saas/models.py

```diff
@@ -1528,6 +1528,26 @@ def provider(invoiced_items):
             result = get_current_provider()
         return result
 
+    def by_organization(self, organization, transaction_type):
```

Comment:
Unless I am mistaken,

        return self.filter(((Q(orig_organization=organization) & Q(orig_account=transaction_type))
            | (Q(dest_organization=organization) & Q(dest_account=transaction_type))))

will return the same subset as

        return self.filter(Q(orig_organization=organization)  | Q(dest_organization=organization)
                       ).filter(Q(orig_account=transaction_type) | Q(dest_account=transaction_type))
      
Both filter() can be split in their own method on a ``QuerySet`` provided we use [Django 1.7 custom manager feature](https://docs.djangoproject.com/en/1.8/releases/1.7/#calling-custom-queryset-methods-from-the-manager).